direct-auth/OktaDirectAuth/OktaDirectAuth/Onboarding/Objects/OnboardingError.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  OnboardingError.swift
+//  OktaDirectAuth
```

Comment:
Please, update the headers across all files :)

Reply:
Grrr, I thought I got all of them. I'll update the copyright template at the workspace level, once I have a workspace configured...